### PR TITLE
Import IP Addresses

### PR DESCRIPTION
**Pull request by jdrew82**
This PR is focused on importing IP Addresses and associated interfaces from CVP. This should address #52.

### CODE REVIEW

**Comment by jdrew82** [comment]
This should be g2g now @qduk.

**Comment by jdrew82** [comment]
Rebased so this comes after the Job update PR.

File: nautobot_ssot_aristacv/tests/fixtures/get_ip_interfaces_client_query.json [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+                        "config",
+                        "ipIntfConfig",
+                        "Loopback2",
+                        "virtualSecondaryWithMask"
+                    ]
+                }
+            }
+        ]
+    }
+]
```
Comment (qduk) [inline nautobot_ssot_aristacv/tests/fixtures/get_ip_interfaces_client_query.json:190]:
```suggestion
]

```

File: nautobot_ssot_aristacv/tests/fixtures/get_ip_interfaces_response.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "interface": "Loopback1",
+        "address": "203.0.113.1/24"
+    },
+    {
+        "interface": "Loopback2",
+        "address": "203.0.113.2/24"
+    }
+]
```
Comment (qduk) [inline nautobot_ssot_aristacv/tests/fixtures/get_ip_interfaces_response.json:10]:
```suggestion
]

```

**Review by qduk — APPROVED** [review]

**Review by qduk — APPROVED** [review]

**Pull request merged**